kaggle/api/kaggle_api_extended.py

```diff
@@ -313,14 +321,17 @@ class KaggleApi(KaggleApi):
   ]
 
   # Competitions valid types
-  valid_competition_groups = ['general', 'entered', 'inClass']
+  valid_competition_groups = [

```

Comment:
If we have Kapigen, why do we need to handcode these things here?

Reply:
Users don't want to type the full-length `kapigen` names, but even if they did we'd still need to verify they were correct. The real reason is for backwards compatibility. The generated names do not match those documented in the API.